### ISSUE TITLE
Two Diff Scenes with Diff Pixel 2D Camera Produces Garbage

### ISSUE THREAD

**Issue by paulhocker** [RyanNielson/PixelCamera2D#2]
It seems like if I have more than one scene that uses pixelcamera2d that the quad textures get messed up and looks like garbage. I am forced to recreate the quad asset again. Do I need to create multiple prefabs? Any ideas?

I have tried creating multiple prefabs, and using the same quad. It just seems to want to do whatever it wants sometimes.

Unity 5.3.2f1 Windows 8.1 64bit

Any help is appreciated.

![snag-0003-3-31-2016-00-31-am](https://cloud.githubusercontent.com/assets/2214652/14166302/e61b011a-f6d7-11e5-8141-e1590f4c564a.png)

**Comment by RyanNielson** [comment]
Any chance you can upload a zip of a simple project that demonstrates this? It seems super strange.

**Comment by paulhocker** [comment]
Yup. I created two diff tests. One with Unity 5.3.2, and one with 5.3.4 - both produce the same results. I really hope this is something dumb I'm doing on my part.

:)
[PixelCamera2D-Corrupt-RenderTexture-Test-PaulHocker-20160331.zip](https://github.com/RyanNielson/PixelCamera2D/files/198987/PixelCamera2D-Corrupt-RenderTexture-Test-PaulHocker-20160331.zip)

**Comment by paulhocker** [comment]
Ryan - in the docs it talks about the fact that the RenderTexture can become corrupt, and to use IsCreated to check. I am looking at your code to see if that is there.

http://docs.unity3d.com/ScriptReference/RenderTexture.html
